section3/reflection.md

```diff
@@ -2,16 +2,89 @@
 
 1. What are two points from the Growth Mindset article and/or video that either resonated with you, or were brand new to you?
 
+- I identify heavily with Carmen's reflection on her fixed mindset before she "hacked a growth mindset." When I was younger, I was quick to quit things that I didn't get right away. I knew that it took struggle and constant practice before anyone gets good at anything but I really hated the idea that other people could see that I was having a hard time learning something and judge me for it. Even coding was something younger me would never have imagined doing because I was convinced that I wasn't a logical person. Shifting into a growth mindset has helped me learn that I've been comparing my progress and process to other people instead of going at a pace that made most sense for me.
+
+- "Can you imagine punishing yourself for every line of code that didn't run properly?" I really love this question from the article because it interrupts my stream of self criticism as I'm learning more about coding. By accepting that mistakes are important and inevitable when I'm learning, it takes the pressure off of getting right the first time and instead helps me slow down and invest time in fully understanding a new concept before moving on.
```

Comment:
Yes!! Failure/Mistakes are part of the process - in fact, I feel like I made my biggest leaps in learning when I tried to find and fix my errors/mistakes :)